ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -587,6 +587,22 @@ DFSS:
   - discharge_fan_run_command
   - discharge_fan_run_status
 
+DF2SS:
```

Comment:
You use supply and discharge in the same abstract. Fix.

Reply:
Modified